astropy/coordinates/tests/test_distance.py

```diff
@@ -117,7 +115,8 @@ def test_distances_scipy():
     from ...cosmology import WMAP5
 
     # try different ways to initialize a Distance
-    d4 = Distance(z=0.23)  # uses default cosmology - as of writing, WMAP7
+    with pytest.warns(RuntimeWarning, match='numpy.dtype size changed'):
+        d4 = Distance(z=0.23)  # uses default cosmology - as of writing, WMAP7
```

Comment:
As in https://github.com/astropy/astropy/pull/7992 I think this is specific to your local installation. I don't think a warning is normally emitted here.

Reply:
Oh, okay!

Reply:
Just to be clear... Should I *not* fix this at all, or revert back to a more generic `catch_warnings`?

Reply:
Ah, ok. Saw your other comment. Removed.